src/ocean/util/app/ext/TimerExt.d

```diff
@@ -27,6 +27,19 @@
 
 module ocean.util.app.ext.TimerExt;
 
+import ocean.util.log.Log;
+
+/*******************************************************************************
+
+    Static module logger
+
+*******************************************************************************/
+
+static private Logger log;
```

Comment:
Redundant static. Also last time a module-level logger was added it didn't go so well. Could you make it class-local?

Reply:
bah, I've copy/pasted it from somewhere else.